src/arrayMethodLastIndexOf.js

```diff
@@ -4,9 +4,24 @@
  * Implement method lastIndexOf
  */
 function applyCustomLastIndexOf() {
-  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex) {
-    // write code here
-  };
+  [].__proto__.lastIndexOf2
+  = function(searchElement, fromIndex = this.length - 1) {
```

Comment:
так не переносять
Ось так краще буде:
```suggestion
  [].__proto__.lastIndexOf2 = function(
    searchElement,
    fromIndex = this.length - 1,
  ) {
```